assets/app/styles/_core.less

```diff
@@ -469,3 +474,8 @@ select:invalid {
 .input-number {
   width: 60px;
 }
+
```

Comment:
shall we make a new .less file for these kinds of overrides?  prefix with `layout.` or `attrs.'?